src/main/java/org/folio/circulation/domain/Request.java

```diff
@@ -197,6 +197,11 @@ public User getRequester() {
     return requester;
   }
 
+  public String getRequesterBarcode(){
+    JsonObject userData = asJson().getJsonObject("requester", new JsonObject());
```

Comment:
please rework this method to work with _representation_ object directly

Reply:
fix